org.eclipse.xtext.xtext.generator/xtend-gen/org/eclipse/xtext/xtext/generator/XtextGenerator.java

```diff
@@ -447,7 +447,29 @@ protected void generatePluginXmls() {
         final IXtextGeneratorFileSystemAccess root = entry.getValue();
         boolean _isFile = root.isFile(pluginXml.getPath());
         if (_isFile) {
-          if ((((!pluginXml.getEntries().isEmpty()) && (!Objects.equal(root.readTextFile(pluginXml.getPath()), pluginXml.getContent()))) && pluginXml.getPath().endsWith(".xml"))) {
+          boolean _and = false;
```

Comment:
I'm wondering why the code is verbose again. With 2.11 Xtend should produce the less verbose code.